composer.json

```diff
@@ -15,7 +15,7 @@
         "ext-json": "*",
         "ext-mbstring": "*",
         "ext-tokenizer": "*",
-        "composer/composer": "^1.7",
+        "composer/composer": "^2.0",
```

Comment:
This should be `^1.7|^2.0`.

Reply:
Can you also check if our test suite will test against both versions of composer?

Reply:
Updated :) What do you mean by testing against both versions of composer ? You want to ensure that current 6 test runs install composer branch 1.x and add 6 other test runs with composer branch 2.x ? How would you achieve that ?

Reply:
Maybe @szepeviktor can help us here?